web/src/pages/project/ProjectGrid.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react'
+import _ from 'lodash'
+import { Table } from 'semantic-ui-react'
+
+import SampleLink from '../../shared/components/links/SampleLink'
+import FamilyLink from '../../shared/components/links/FamilyLink'
+import sanitiseValue from '../../shared/utilities/sanitiseValue'
+import { ProjectSummaryResponse } from '../../sm-api/api'
+import MuckError from '../../shared/components/MuckError'
+
+interface ProjectGridProps {
+    summary?: ProjectSummaryResponse
+    projectName?: string
+    error?: string
+}
+
+const ProjectGrid: React.FunctionComponent<ProjectGridProps> = ({
+    summary,
+    projectName,
+    error,
+}) => {
+    if (!projectName) {
```

Comment:
I would change condition to `!projectName || Object.keys(summary ?? {}).length === 0` if following the comment below, to check for an empty summary data object.